components/comparisons/comparisonTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { Container, Row, Col, Table } from "react-bootstrap";
+import Image from "../Image";
+import ComparisonRow from "./comparisonRow";
+
+export default function ComparisonTable({ comparisonData }) {
+  // comparisonData = [{
+  // category: [{
+  //   feature: featureName,
+  //   grouparoo: true,
+  //   comp: false
+  // },]
+  // }]
```

Comment:
If you did want to keep this information around, you could make a typescript type, and then say that the params you were getting into the component were of that type:

```ts
import {ComparisonData} from '../utils/comparisonPages'

export default function ComparisonTable({ comparisonData } : {comparisonData: ComparisonData}) {
  // ...
}

```

Reply:
As a workflow question.... I agree these changes should be made, but you and Andy approved it as is... do I make the changes then squash and merge?  OR just squash and merge and tidy those up in the next iteration (low-stakes here as it's just removing unneeded comments...)